extensions/amp-analytics/0.1/activity-impl.js

```diff
@@ -119,6 +123,15 @@ const ACTIVE_EVENT_TYPES = [
   'mousedown', 'mouseup', 'mousemove', 'keydown', 'keyup',
 ];
 
+/**
+ * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampDoc
+ * @return {Activity}
+ */
+export function activityServiceForTesting(ampDoc) {
```

Comment:
Can this method just install the service (`installActivityService`)? Seems like returning the service isn't necessary in current usage.

Aside: are you on Hangouts or Slack today?